Makefile

```
@@ -29,6 +29,9 @@ STOP_SVCS = $(shell tac .services | grep -v \\\#)
 # List of hosts available in devenv
 HOSTS_LINES = $(shell grep -Rl IPV4_PREFIX ./services/* | grep .hosts)
 
+# Paths to protocol.privnet.yml
+MORPH_CHAIN_PROTOCOL = $('./services/morph_chain/protocol.privnet.yml')
```

Do you need `$('...')` there? Seems it is simple constants so you can
```suggestion
MORPH_CHAIN_PROTOCOL = './services/morph_chain/protocol.privnet.yml'
```

---

Done